.github/workflows/upload-delegates.yml

```diff
@@ -0,0 +1,30 @@
+on: [pull_request]
```

Comment:
Only people with write access to the community repo are able to make PRs? Or anyone can?

Reply:
i think everyone can.. good point, maybe it should be limited to master branch when merged?

Reply:
Well I was thinking that we could also automatically update the ens text record in the github actions, in which case we'd definitely need this to be locked down to people with the right permissions.

Or do you think it'd be better to have someone manually review things before manually updating the ens record?